### ISSUE TITLE
Table unusable on v1.56.0

### ISSUE THREAD

**Issue by chrissantamaria** [mbrn/material-table#1569]
**Describe the bug**

material-table is unusable on the latest version 1.56.0. When attempting to use the main `MaterialTable` component, I get the following error:

```
DependencyNotFoundError
Could not find dependency: 'material-ui-pickers' relative to '/node_modules/material-table/dist/components/m-table-edit-field.js'
```

**To Reproduce**

Try the [Getting Started example](https://material-table.com/#/docs/get-started). CodeSandbox available [here](https://codesandbox.io/s/material-table-dependency-bug-l4102).

**Expected behavior**

The table renders properly.

**Desktop (please complete the following information):**

- OS: macOS Catalina
- Browser: Chrome
- Version 79.0.3945.117

Like mentioned above, this also occurs in an isolated CodeSandbox, so it's likely not a local dependency issue.

**Additional context**

Rolling back to v1.55.0 in the CodeSandbox fixes the issue, suggesting this is a new bug.

Viewing `material-table/dist/components/m-table-filter-row.js` shows the following line from the Babel output:

```js
var _materialUiPickers = require('material-ui-pickers');
```

Shouldn't this be importing from `@material-ui/pickers`? Not sure where `material-ui-pickers` is coming from - doing a search of it in the repo doesn't have any results.

**Comment by mbrn** [comment]
Hi @chrissantamaria ,


I am looking it. Let me resolve it.

**Comment by mbrn** [comment]
I solved it with 1.56.1
